src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -12,11 +12,12 @@ At the end of the conversion, a sanitizing operation is started on the final
 model. See [sanitizer.md](sanitizer.md) for more information.
 
 ### Prepending data
-The NTFS format introduce 2 objects to enable the manipulation of several datasets : contributors and datasets. Those two objects are not described here. The construction of NTFS objects IDs requires, for uniqueness purpose, that a unique prefix (specified for each source of data) needs to be included in every object's id.
-
+As explained in [common.md](common.md), a prefix is added to all identifiers during the conversion in order to guarantee uniqueness among objects IDs.
 In the following chapters, identifiers may be prepend with this _prefix_ using this pattern : **\<prefix>:<object\_id>**.
 The use of this specific pattern is shown explicitly using the value **ID** in the column _Constraint_ in the tables below.
 
+In addition, the NTFS format introduces 2 objects to enable the manipulation of several datasets : contributors and datasets. Those two objects are not described here, see [common.md](common.md) for more information. 
```

Comment:
Sorry, I have an OCD :stuck_out_tongue_closed_eyes:... No space before `:` in English. 
```suggestion
In addition, the NTFS format introduces 2 objects to enable the manipulation of several datasets: contributors and datasets. Those two objects are described in [common.md](common.md). 
```